examples/classic3.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Bumble HID Keyboard",
```

Comment:
This file had better to be named `hid_keyboard.json`

Reply:
Its changed to hid_keyboard.json